workspace/2023/12/2023-12-numfocus-update.md

```diff
@@ -0,0 +1,24 @@
+## Conda
+
+### Community 
+
+* Subscribe to the new [conda-announce mailing list](https://lists.conda.org/wws/subscribe/announce) to get occasional summaries on the conda ecosystem delivered to your inbox.
+* The Mamba project has [bi-weekly community calls](https://hackmd.io/iOgkxNMpTea6lWczjCov9Q?view) on Tuesdays.
+* The overall conda community has [bi-weekly calls](https://hackmd.io/@conda-community?tags=%5B%22meeting-notes%22%5D) on Wednesdays. 
+
+**Releases**
+
+* **`conda`** [23.11.0 release](https://conda.org/blog/2023-12-07-november-releases/#changes-in-conda-23110) includes 
```

Comment:
This makes it sound like all these things happened in the same release. What about:

```suggestion
* **`conda`** [23.11.0](https://conda.org/blog/2023-12-07-november-releases/#changes-in-conda-23110) closes the 23.x release series. 2023 has brought many features to `conda`, including: 
```